core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaUpdateRecipesTranslator.java

```diff
@@ -427,7 +448,7 @@ List<String> translateShulkerBoxRecipe(GeyserShapelessRecipe recipe) {
                 return null;
             }
             // Strip NBT - tools won't appear in the recipe book otherwise
-            output = output.toBuilder().tag(null).build();
+            // output = output.toBuilder().tag(null).build(); // TODO confirm???
```

Comment:
Did you accidentally comment out the whole line? otherwise I am reading your PR description wrong haha

Reply:
Nope, this is intentional - Bedrock wants the tag for firework crafting, and crafting tools so far seems to work.. but it's a to-do as I couldn't replicate the original reason for why this workaround is needed